src/web/src/pages/_document.tsx

```diff
@@ -20,6 +20,20 @@ class MyDocument extends Document {
           <link rel="icon" href={logoUrl} type="image/svg+xml" />
           <meta charSet="utf-8" />
           <meta name="theme-color" content={lightTheme.palette.primary.main} />
+          <link rel="manifest" href="/manifest.json" />
+          <link rel="apple-touch-icon" sizes="192x192" href="/logoImages/manifest-icon-192.png" />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="192x192"
+            href="/logoImages/manifest-icon-192.png"
+          />
+          <link
+            rel="shortcut icon"
```

Comment:
We have a hook that manages the icon and adds a red dot when there are new posts.  We should make sure this won't break anything with what we do in https://github.com/Seneca-CDOT/telescope/blob/master/src/web/src/hooks/use-favicon-badge.ts#L14.

Reply:
@humphd I think I will get rid of these links and only leave the `apple-touch-icon` link